docs/layouts/base.css

```diff
@@ -20,6 +20,44 @@
   position: fixed;
   width: 50vw;
   height: 100vh;
-  background-color: #f6f8fa;
+  background-color: var(--shade0);
   z-index: -1;
 }
+
+.rightBackground {
+  position: fixed;
+  width: 50vw;
+  height: 100vh;
+  right: 0;
+  top: 0;
+  background-color: var(--shade1);
+  z-index: -1;
+}
+
+:global .numberTitle {
+  display: inline-block;
+  height: 30px;
+  width: 30px;
+  line-height: 30px;
+  -moz-border-radius: 30px;
+  border-radius: 30px;
+  background-color: var(--primary);
+  color: white;
+  text-align: center;
+  font-weight: 600;
+}
+
+:not(pre) > code {
+  font-family: "Roboto Mono", monospace;
+  font-size: 15px;
+  -webkit-border-radius: 3px;
+  -moz-border-radius: 3px;
+  border-radius: 5px;
+  -moz-background-clip: padding;
+  -webkit-background-clip: padding-box;
+  background-clip: padding-box;
```

Comment:
I feel like our sass compiler may be smart enough to add prefixing. Maybe I'm wrong though. It would be nice if we could add a preprocessor to add prefixes if not.